ecl/resource_ecl_vna_appliance_v1_mock_test.go

```diff
@@ -427,9 +429,16 @@ resource "ecl_vna_appliance_v1" "appliance_1" {
 		ip_address = "192.168.1.50"
 	}
 
+    initial_config = {
+        format = "set"
+        data = "c2V0IGludGVyZmFjZXMgZ2UtMC8wLzAgZGVzY3JpcHRpb24gc2FtcGxl"
+    }
+
 	lifecycle {
 		ignore_changes = [
 			"default_gateway",
+            "username",
+            "password",
```

Comment:
https://github.com/nttcom/terraform-provider-ecl/pull/101/commits/f257e76328b397d5cc302558bf3b523a61cfdc01

にてmock testが通らなくなっていたので追従

Reply:
[MUST] スペースではなくタブでインデントしてください
(他の箇所も同様に修正をお願いします)

```suggestion
		"username",
		"password",
```

Reply:
修正します(goの拡張入れます)